app/src/test/java/com/tvtracker/IntegrationTests.kt

```diff
@@ -42,7 +42,8 @@ class IntegrationTests {
 
 
     @Test
-    fun `Given media data is available when I search BattleBots then I should receive media results` () = runTest {
+    fun `Given media data is available when I search BattleBots then I should receive media results`
+                () = runTest {
```

Comment:
Code has a column limit of 100 characters 
Exceptions Include:
- Lines where obeying the column limit is not possible (for example, a long URL in KDoc)
- package and import statements
- Command lines in a comment that may be cut-and-pasted into a shell
https://developer.android.com/kotlin/style-guide